### DISCUSSION TITLE
How to use BlurTransformation.kt in version 2.1.0

### DISCUSSION THREAD

**Discussion by gaohomway** [coil-kt/coil#1378] [Q&A] [answered]
I am using version 2.1.0, but the BlurTransformation has been removed. The documentation says that copying the file can be imported, but I have a lot of errors when I copy the code to my project. I don't know what package to import.

<img width="985" alt="截屏2022-07-07 19 23 19" src="https://user-images.githubusercontent.com/5456137/177762290-5a08dfa4-4446-4c19-a86b-2aee8025c8ee.png">

**Comment by colinrtwhite** [comment, marked as answer]
Already answered here (with the same title too 🤔): https://github.com/coil-kt/coil/discussions/1362
